model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/JpaUpdate22_0_0_RemoveRhssoThemes.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.connections.jpa.updater.liquibase.custom;
+
+import liquibase.exception.CustomChangeException;
+import liquibase.statement.core.DeleteStatement;
+import liquibase.statement.core.UpdateStatement;
+import liquibase.structure.core.Table;
+import org.keycloak.theme.DefaultThemeSelectorProvider;
+
+/**
+ * <p>Migration class to remove old <em>rh-sso</em> themes for login and account.</p>
+ *
+ * @author rmartinc
+ */
+public class JpaUpdate22_0_0_RemoveRhssoThemes extends CustomKeycloakTask {
+
+    @Override
+    protected void generateStatementsImpl() throws CustomChangeException {
+        // remove login theme for realms
+        statements.add(new UpdateStatement(null, null, database.correctObjectName("REALM", Table.class))
+                .addNewColumnValue("LOGIN_THEME", null)
+                .setWhereClause("LOGIN_THEME=?")
+                .addWhereParameter("rh-sso"));
+        // remove account theme for realms
+        statements.add(new UpdateStatement(null, null, database.correctObjectName("REALM", Table.class))
```

Comment:
Just looked at the RH-SSO 7.6.3 and seeing that "email" theme has also "rh-sso" as available value. So looks that one should be also modified the same way? It was not mentioned in the description of the GH issue, but now figured that we have it according to this screenshot.

AFAIK admin theme is not needed as it was already done in `MigrateTo21_0_0`

Reply:
Yes, email and admin also have rh-sso version. The admin one was already changed by a previous upgrade but email is not managed. Reading the description I thought it was not needed. Okis, I'll add another update sentence then for email.

Reply:
Okis! Done!

Reply:
Thanks!